.changeset/flat-bananas-hug.md

```diff
@@ -0,0 +1,5 @@
+---
+"stylelint": patch
+---
+
+Fixed: `selector-type-no-unknown` false positives
```

Comment:
[suggest] Just to be sure, does clarifying targets of this change make sense? E.g.

```suggestion
Fixed: `selector-type-no-unknown` false positives for deprecated HTML tags
```